packaging/requirements/requirements-azure.txt

```diff
@@ -4,7 +4,7 @@ azure-cli-core==2.0.35
 azure-cli-nspkg==3.0.2
 azure-common==1.1.11
 azure-mgmt-batch==4.1.0
-azure.mgmt.cdn==3.0.0
+azure-mgmt-cdn==3.0.0
```

Comment:
so is it really necessary? as it somehow worked with broken requirement....

Reply:
i think it's necessary to do right thing. actually i didn't understand why previous version works :(

Reply:
so are you sure we need this dependency?

Reply:
oh, perhaps correct version is in test requirements..... but then sanity tests wouldn't pass....

Reply:
yes, I think you also need to update in test/runner/requirements

Reply:
nice catch. fixed. the dependency is for cdn.